mux.go

```diff
@@ -92,6 +92,11 @@ type routeConf struct {
 	buildScheme string
 
 	buildVarsFunc BuildVarsFunc
+
+	// If true, if the url is cleaned, will send
```

Comment:
RE: this - 

* How do you know the URL is cleaned? (note: uppercase "URL")
* Why is this useful to a user to enable, if they haven't read this PR or the history?